src/firmware-update.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: base
+description: Zigbee and Thread USB stick by the creators of Home Assistant
+---
+
+<div class="container">
+  <div>
+    <h1>Firmware update</h1>
+    <p>First things first: a firmware upgrade is not required for SkyConnect at this point.</p>
+    <p>However: We're working on making firmware updates available via Home Assistant.
+      These updates are still under development.
+    </p>
+  </div>
+  <div><h3 class="text-center">SL Web Tools to install firmware updates</h3></div>
+  <p>
+    If you're feeling adventurous and want to try out the latest (experimental) firmware update, you can use
+    the <a href="https://www.home-assistant.io/blog/2023/02/08/state-of-matter-and-thread/#silabs-multi-flasher--sl-web-tools" target="_blank">web-flasher</a> below to install the firmware directly from your web-browser over USB.
```

Comment:
That link can be confusing, as this is the source code of the flasher below. I think I'd remove it here, and maybe add a sentence after the updater, e.g.:

"This firmware updater has been built using SL Web Tools." (link SL Web Tools to that site).

Reply:
The link points to the blog article _State of Matter and Thread_ which axplains more about the SL Web Tools.

Reply:
Right, but it is/was still kinda confusing in the sense do I have to follow the link to do the update?

Reply:
I understand. I completely reworded the section now. I like the link there, as the blog gives a bit more context. Would you mind taking another look, @agners?